Cargo.toml

```diff
@@ -10,11 +10,17 @@ members = [
     "crates/fj-math",
     "crates/fj-operations",
     "crates/fj-viewer",
+    "crates/fj-proc",
+
     "crates/fj-window",
 
+    "crates/fj-proc",
+
+
```

Comment:
There's the same crate a second time. I'm surprised that doesn't result in some kind of error! :smile: